Pod/Classes/SEGMixpanelIntegration.m

```diff
@@ -139,6 +140,26 @@ - (void)screen:(SEGScreenPayload *)payload
     }
 }
 
+
+- (void) group:(SEGGroupPayload *)payload {
+    
+    NSString *groupID = payload.groupId;
+    NSDictionary *traits = [payload traits];
+    NSString *groupName = traits[@"name"];
```

Comment:
In the a.js implementation we `return` if there is no `groupId` or `groupIdentifierTraits`. Can we implement that here as well?